servers/rendering/rasterizer_rd/shaders/giprobe.glsl

```diff
@@ -51,7 +51,7 @@ struct Light {
 	float attenuation;
 
 	vec3 color;
-	float spot_angle_radians;
+	float spot_angle;
```

Comment:
If these are cos spot angle, would they be better named cos_spot_angle? Just a thought if it made reading easier later on when someone comes back to this.